src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -29,14 +30,14 @@ const FILE_TYPES = [
   'jpg',
   'bmp'
 ];
+
 const MIN_SIZE = 350;
+const MIN_RATIO = 0.2;
+const MAX_RATIO = 1.7;
+const WARN_RATIO = 1.3;
```

Comment:
Could we test this with images created by selfie cameras on phones and laptop cameras? I wouldn't mind doing the testing on a phone if it's just a matter of running it locally.